stateproof/db.go

```diff
@@ -86,6 +124,24 @@ func getPendingSigs(tx *sql.Tx) (map[basics.Round][]pendingSig, error) {
 	return rowsToPendingSigs(rows)
 }
 
+func getPendingSigsForRound(tx *sql.Tx, rnd basics.Round) ([]pendingSig, error) {
+	rows, err := tx.Query("SELECT sprnd, signer, sig, from_this_node FROM sigs WHERE sprnd=?", rnd)
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+	tmpmap, err := rowsToPendingSigs(rows)
+	if err != nil {
+		return nil, err
+	}
+	signaturesAtRound, exists := tmpmap[rnd]
+	if exists {
+		return signaturesAtRound, nil
+	}
+	return nil, nil
```

Comment:
should avoid `signaturesAtRound, exists := tmpmap[rnd]`. 
use: `return tmpmap[rnd], nil`